_2020/data-wrangling.md

```diff
@@ -332,7 +340,7 @@ ssh myserver journalctl
 R is another (weird) programming language that's great at data analysis
 and [plotting](https://ggplot2.tidyverse.org/). We won't go into too
 much detail, but suffice to say that `summary` prints summary statistics
-about a matrix, and we computed a matrix from the input stream of
+for a vector, and we created a vector containing the input stream of
```

Comment:
Why correct matrix to vector here?

Reply:
Because I believe you're creating a vector there, not a matrix. R is weird :)

A slightly modified example, taken from my dictionary exercise solutions:
<img width="1078" alt="Screen Shot 2020-06-01 at 2 41 05 PM" src="https://user-images.githubusercontent.com/1282353/83442325-faf74900-a415-11ea-900f-e5882b577eb3.png">

Reply:
R is weird indeed :p